recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import ConanFile, tools
+from pathlib import Path
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch": ["x86_64"],
+        "os": [
+            "Linux",
+            "Macos",
+            "Windows"
+        ]
+    }
+    no_copy_source=True
+
+    def source(self):
+        tools.get(**self._data['sources'])
+
+    def build(self):
+        tools.download(**self._data['installer'], filename = "creator.7z")
+        self.run("cmake -E tar x creator.7z")
```

Comment:
Add `7zip/19.00@` as build requirement. 7z is not a default app.

Reply:
This uses the 7z module that is baked into cmake which works on all platforms. `Add 7zip/19.00@` is only available for Windows.